server.go

```diff
@@ -2543,6 +2577,42 @@ func (s *Server) processor(client MessagingClient, done chan struct{}) {
 	}
 }
 
+// flusher sends raw point data in batches to the broker by topic.
+func (s *Server) flusher(client MessagingClient, done chan struct{}) {
+	ticker := time.NewTicker(s.FlushInterval)
```

Comment:
A `Ticker` will always fire every `x` duration whereas an `After` will fire every `x+elapsed` duration (where `elapsed` is the time taken between calls to `After`).